pkg/ui/src/views/reports/containers/debug/index.tsx

```diff
@@ -79,285 +80,399 @@ function DebugPanelLink(props: { name: string, url: string,  note: string }) {
   );
 }
 
-export default function Debug() {
-  return (
-    <div className="section">
-      <Helmet title="Debug" />
-      <h1 className="base-heading page-title">Advanced Debugging</h1>
-      <div className="debug-header">
-        <InfoBox>
-          <p>
-            The following pages are meant for advanced monitoring and troubleshooting.
-            Note that these pages are experimental and undocumented. If you find an issue,
-            let us know through{" "}
-            <a href={ COMMUNITY_URL }>these channels.</a>
-          </p>
-        </InfoBox>
+interface IDebugProps {
+  nodesSummary: NodesSummary;
+  dataSource: DecommissionedNodeStatusRow[];
+}
+
+interface IDebugState {
+  selected: string;
+  location: string;
+  opened: boolean;
```

Comment:
State like this should live in the extracted dropdown component. It doesn't make sense for `Debug` to have an `opened` state since the open refers to the dropdown, not debug.